card/spec/lib/card/reference_spec.rb

```diff
@@ -170,8 +172,8 @@
       name: 'search with references',
       content: '{"name":"X", "right_plus":["Y",{"content":["in","A","B"]}]}'
     )
-    y_referencers = Card['Y'].referencers.map &:name
-    expect(y_referencers).to include('search with references')
+    y_referers = Card['Y'].referers.map &:name
```

Comment:
Ambiguous block operator. Parenthesize the method arguments if it's surely a block operator, or add a whitespace to the right of the `&` if it should be a binary AND.